### PR TITLE
Create ViewWorkoutPlanButton and add to Settings + Create ViewWorkoutPlanScreen

### PR DESCRIPTION
**Pull request by umjennifer**
- Create ViewWorkoutPlanScreen to view current (active) workout plan  
- Create ViewWorkoutPlanButton and add to Settings
- Add TODOs for future considerations and updates
- Upper case "out" in Sign Out button

### CODE REVIEW

File: myFitnessTrainer/src/screens/ViewWorkoutPlanScreen.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+const ViewWorkoutPlanScreen = ({route}) => {
+    const { duration, fitnessGoal, fitnessLevel, startDate, daysCompleted, workoutsPerDay } = route.params;
+    const formattedStartDate = new Date(startDate).toDateString();
+    let formattedExercisesPerDay = []
+    for (const dayNum in workoutsPerDay){
+        formattedExercisesPerDay.push(<Text>Day {dayNum}{"\n"}</Text>);
+        for (const exerciseNum in workoutsPerDay[dayNum]){
+            formattedExercisesPerDay.push(<Text>{workoutsPerDay[dayNum][exerciseNum]}{"\n"}</Text>);
+        }
+        formattedExercisesPerDay.push(<Text>{"\n"}</Text>)
```
Comment (ethorne2) [inline myFitnessTrainer/src/screens/ViewWorkoutPlanScreen.js]:
```suggestion
    for (const dayNum in workoutsPerDay){
        formattedExercisesPerDay.push(<Text key={dayNum}>Day {dayNum}{"\n"}</Text>);
        for (const exerciseNum in workoutsPerDay[dayNum]){
            formattedExercisesPerDay.push(<Text key={dayNum.concat("-", exerciseNum)}>{workoutsPerDay[dayNum][exerciseNum]}{"\n"}</Text>);
        }
        formattedExercisesPerDay.push(<Text>{"\n"}</Text>)
```

Comment (ethorne2) [reply myFitnessTrainer/src/screens/ViewWorkoutPlanScreen.js]: suggesting this update for a warning about items in a list not having unique keys.

Comment (umjennifer) [reply myFitnessTrainer/src/screens/ViewWorkoutPlanScreen.js]: Thanks, good catch! Updated

**Review by ethorne2 — APPROVED** [review]
🚀

**Review by ethorne2 — APPROVED** [review]
🚀

**Pull request merged**